carts/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+
+from carts.models import Cart
+from users.models import User
+from products.models import Product, Size
+from users.utils import login_deco
+
+class CartView(View):
+    @login_deco
+    def post(self, request):
+        data = json.loads(request.body)
+
+        try:
+            if not Product.objects.filter(id = data["product_id"]).exists():
+                return JsonResponse({"message":"PRODUCT_DOES_NOT_EXIST"}, status = 400)
+            
+            if not Size.objects.filter(id = data["size_id"]).exists():
+                return JsonResponse({"message":"SIZE_DOES_NOT_EXIST"}, status = 400)
+
+            if data["count"] == 0:
+                return JsonResponse({"message":"INVALID_COUNT"}, status = 400)
+
+            user     = User.objects.get(id = 5)
+            product  = Product.objects.get(id = data["product_id"])
+            size     = Size.objects.get(id = data["size_id"])
+
+            cart, is_created = Cart.objects.get_or_create(user = user, product = product, size = size)
+            
+            if is_created:
+                cart.count = 0
+            
+            cart.count += data["count"]
+            cart.save()
+                
+            return JsonResponse({"message":"CREATED"}, status = 201)
```

Comment:
```suggestion
            return JsonResponse({"message":"SUCCESS"}, status = 200)
```
장바구니가 생성되거나 추가되거나 하는 상황이 모두 있으므로 200으로 통일해도 좋을 것 같습니다.

Reply:
200으로 통일했습니다